client/lib/csv/index.js

```diff
@@ -0,0 +1,65 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import moment from 'moment';
+import { saveAs } from 'browser-filesaver';
+
+function getCSVHeaders( headers ) {
+	return headers.map( header => header.label ).join( ',' );
```

Comment:
Do we need to ensure we are working with an array as an argument here and in `getCSVRows`, `getCSVTotals`? Maybe we return an empty array if the argument is null or not an array?

Reply:
Ah I see `generateCSVDataFromTable` adds array defaults, but still wonder if we need to validate the inputs are indeed arrays here.